flutter/android/src/main/kotlin/io/sentry/flutter/SentryFlutterPlugin.kt

```diff
@@ -87,7 +88,7 @@ class SentryFlutterPlugin : FlutterPlugin, MethodCallHandler {
         options.isEnableAppComponentBreadcrumbs = it
       }
       args.getIfNotNull<Int>("maxBreadcrumbs") { options.maxBreadcrumbs = it }
-      args.getIfNotNull<Int>("cacheDirSize") { options.cacheDirSize = it }
+      args.getIfNotNull<Int>("cacheDirSize") { options.maxCacheItems = it }
```

Comment:
```suggestion
      args.getIfNotNull<Int>("maxCacheItems") { options.maxCacheItems = it }
```

we should fix the param name passing as well, also on iOS which is already called maxCacheItems I guess

Reply:
I would rather do it as part of https://github.com/getsentry/sentry-dart/issues/485

Reply:
sounds good